Library/Formula/libglademm.rb

```diff
@@ -1,9 +1,8 @@
-require 'formula'
-
 class Libglademm < Formula
-  homepage 'http://gnome.org'
-  url 'http://ftp.gnome.org/pub/GNOME/sources/libglademm/2.6/libglademm-2.6.7.tar.bz2'
-  sha1 'd7c0138c80ea337d2e9ae55f74a6953ce2eb9f5d'
+  homepage "https://gnome.org"
+  url "http://ftp.gnome.org/pub/GNOME/sources/libglademm/2.6/libglademm-2.6.7.tar.bz2"
```

Comment:
For this one too:
https://download.gnome.org/sources/libglademm/2.6/libglademm-2.6.7.tar.bz2